run.go

```diff
@@ -36,6 +37,11 @@ import (
 	"github.com/hazelcast/hazelcast-commandline-client/types/mapcmd"
 )
 
+const (
+	ViridianCoordinatorURL    = "https://api.viridian.hazelcast.com"
+	HzCloudCoordinatorBaseUrl = "HZ_CLOUD_COORDINATOR_BASE_URL"
```

Comment:
Could you make `Url` all-uppercase?

Reply:
How about EnvHzCloudCoordinatorBaseURL ?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/145/commits/8738eaeded94567646916d6707a5ad0a089612a8